fastlane/Fastfile

```diff
@@ -120,8 +120,7 @@ platform :android do
       gradle(
         project_dir: 'Mobile-Expensify/Android',
         task: 'assemble',
-        flavor: 'Production',
```

Comment:
This seems _fine_, but I wonder if we can do an unsigned release build?

Reply:
I don't think we can without keys